### PR TITLE
Pd working on menu

### PR DESCRIPTION
**Pull request by PatrickDolanJ**
Main menu and How To Play done ENOUGH

### CODE REVIEW

**Comment by PatrickDolanJ** [comment]
Added logo Splash screen, slowed down rotation in slow mo mode, "finished" how to play and menu

File: Assets/Scenes/Hilly plane level 1.unity [hunk 1]
```diff
@@ -346,6 +346,149 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: fe87c0e1cc204ed48ad3b37840f39efc, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!1 &467880823
```
Comment (danpprince) [inline Assets/Scenes/Hilly plane level 1.unity]: Looks like there were some changes to only this scene? Kind of suspicious, maybe it was meant to be applied to other scenes too or was committed on accident?

**Comment by danpprince** [comment]
Filename spelling: `Assets/Sprites/falg_bigger_border.png` -> _flag_ _bigger_border.png

File: Assets/Scenes/Hilly plane level 1.unity (hunk 1 above)
Comment (PatrickDolanJ) [reply Assets/Scenes/Hilly plane level 1.unity]: Yes there was a missing commit, got it up now should be good to go.

**Comment by danpprince** [comment]
~Currently every other level has music vs is silent, must be some bug on toggling that state.~ should by good after my latest commit

**Review by danpprince — APPROVED** [review]
Looks good! Play tested as well

**Pull request merged**